File: src/main/java/com/server/ordering/Properties.java
package com.server.ordering;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationProperties("env")
@Getter @Setter
public class Properties {

    // twilio
    private String account_sid;
    private String auth_token;
    private String path_service_sid;
}

File: src/main/kotlin/com/byeoru/ordering_server/Properties.kt
package com.byeoru.ordering_server

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationProperties("env")
class Properties {
    // twilio
    var account_sid: String? = null
    var auth_token: String? = null
    var path_service_sid: String? = null
}